src/actions/preview.js

```diff
@@ -103,7 +103,7 @@ export function updatePreview(target: HTMLElement, editor: any) {
     if (preview) {
       // Return early if we are currently showing another preview or
       // if we are mousing over the same token as before
-      if (preview.updating || isEqual(preview.tokenPos, tokenPos)) {
+      if (isEqual(preview.tokenPos, tokenPos)) {
```

Comment:
`updatePreview` was never added to the unit tests.
https://github.com/devtools-html/debugger.html/blob/master/src/actions/tests/preview.spec.js

Could it be possible to add this as a test so to prevent future regressions?

I'd be happy to mentor with that.